addons/mail/static/src/models/emoji_list.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many, one } from '@mail/model/model_field';
+import { insertAndReplace, replace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'EmojiList',
+    identifyingFields: ['emojiPickerView'],
+    recordMethods: {
+        onComponentUpdate() {
+            if (this.component && this.component.root.el) {
+                this.component.root.el.height = this.height;
+            }
+        },
```

Comment:
This would be worth a comment because I don't understand why we need to restore height like that, the current model doesn't even have a height field?

Reply:
Actually no longer needed this code. It's been put in template instead